pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -1256,52 +1256,6 @@ func TestValidateAllowPrivilegeEscalation(t *testing.T) {
 	if len(errs) != 0 {
 		t.Errorf("directly allowing privilege escalation expected no errors but got %v", errs)
 	}
-}
-
-// TestValidateDefaultAllowPrivilegeEscalation will test that when the podSecurityPolicy
-// DefaultAllowPrivilegeEscalation is false we cannot set a container's
-// securityContext to allowPrivilegeEscalation but when it is true we can.
-func TestValidateDefaultAllowPrivilegeEscalation(t *testing.T) {
```

Comment:
I'm finding these tests very hard to reason about... what do you think of this instead? https://github.com/kubernetes/kubernetes/pull/68105

Reply:
I like the table test replacement